src/validate.js

```diff
@@ -99,6 +112,31 @@ function unwrapErrorHandler(errorHandler) {
 }
 
 /**
+ * Transforming the error output from ajv to format used by jsonschema.
+ * At some point, components should be updated to support ajv.
+ */
+function transformAjvErrors(errors = []) {
+  if (errors === null) {
+    return [];
+  }
+
+  return errors.map(e => {
+    const { dataPath, keyword, message, params } = e;
+    // not having 'instance' in the property name breaks things further down the line.
+    let property = `instance${dataPath}`;
```

Comment:
Could you elaborate on this?

Reply:
Not having 'instance' in the property value caused an additional 8 tests to fail so I was thinking that there's a dependency in some of the components for this, and didn't dig further. Maybe I just need to update the tests. I'll take a closer look at the tests.

Reply:
I pushed another commit to remove the reliance on `instance` and updated the affected tests.